src/App.css

```diff
@@ -13,11 +13,24 @@
   justify-content: space-evenly;
 }
 
+main {
+  display: grid;
+  grid-template-areas:
+    'canvas desc'
+    'canvas editor';
+  gap: 1rem;
```

Comment:
this does not need to be relative I think

Reply:
r stands for root

Reply:
its better if it scales with the device, not font size in .app

Reply:
ill remove it and fix it with `space-evenly` instead